src/api/graasp.js

```diff
@@ -2,7 +2,8 @@ const qs = require('qs');
 
 // app routes and parameters
 export const RESEARCH_API_ROUTE = 'research';
-export const USERS_API_ROUTE = 'users';
+export const ITEM_API_ROUTE = 'item';
```

Comment:
For consistency I would use `items`. I don't know if you have more constraint for it.

Reply:
Changed. I have to change also the endpoint in Graasp-Plugin-Actions.